### PR TITLE
✨Add motor stop function

### PR DESCRIPTION
**Pull request by omegaStag**
#### Summary:
Adds a function to set the motor voltage to zero.

#### Motivation:
There isn't already a motor stop function

#### Test Plan:
Make motor run and use function to stop motor

### CODE REVIEW

**Review by WillXuCodes — APPROVED** [review]
Looks good

**Review by HotelCalifornia — CHANGES_REQUESTED** [review]
I thought the point of this was to add a function that stops a motor respecting the set brake mode? as-is this PR doesn't do that. instead of internally calling move_voltage(0) it should call e.g. move_velocity(0).

also i'd recommend calling it motor_brake/pros::Motor::brake instead of motor_stop/pros::Motor::stop to make it clear that's what's happening.

File: include/pros/motors.h [hunk 1]
```diff
@@ -56,6 +56,22 @@ namespace c {
  */
 int32_t motor_move(uint8_t port, int32_t voltage);
 
+/**
+ * Sets the velocity for the motor to 0, stopping the motor with respect to set brake mode.
```
Comment (HotelCalifornia) [inline include/pros/motors.h:60]:
```suggestion
 * Stops the motor using the currently configured brake mode.
 * 
 * This function sets motor velocity to zero, which will cause it to act
 * according to the set brake mode. If brake mode is set to MOTOR_BRAKE_HOLD,
 * this function may behave differently than calling motor_move_absolute(port, 0)
 * or motor_move_relative(port, 0).
```

File: include/pros/motors.hpp [hunk 2]
```diff
@@ -185,6 +185,18 @@ class Motor {
 	 */
 	virtual std::int32_t move_voltage(const std::int32_t voltage) const;
 
+	/**
+	 * Sets the velocity for the motor to 0, stopping the motor with respect to set brake mode.
```
Comment (HotelCalifornia) [inline include/pros/motors.hpp:189]:
```suggestion
	 * Stops the motor using the currently configured brake mode.
	 *
	 * This function sets motor velocity to zero, which will cause it to act
	 * according to the set brake mode. If brake mode is set to MOTOR_BRAKE_HOLD,
	 * this function may behave differently than calling move_absolute(0)
	 * or move_relative(0).
```

**Review by HotelCalifornia — COMMENTED** [review]
some updates and clarifications I thought would be useful in the documentation

**Review by WillXuCodes — APPROVED** [review]
Works on hardware, feel free to merge later tonight nearer to 3.6 release

**Pull request merged**